### PR TITLE
feat: run local using docker

### PR DESCRIPTION
**Pull request by mhalambek**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ENV RUSTUP_HOME=/usr/local/rustup \
+    CARGO_HOME=/usr/local/cargo \
+    PATH=/usr/local/cargo/bin:$PATH
+
+RUN curl https://sh.rustup.rs -sSf | \
+    sh -s -- -y --no-modify-path --default-toolchain "$(cat /tmp/rust-toolchain)"
+
+RUN cargo install diesel_cli --no-default-features --features "postgres"
+COPY . .
+RUN cargo build --release -p indexer-explorer
```
Comment (frol) [inline Dockerfile:28]: I would create a second stage extracting the binary itself, and the nearcore home folder to greatly slim down the image

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -1,12 +1,8 @@
...
-    label: "Formatter"
-    
-  - command: "source ~/.cargo/env && cargo clippy"
-    label: "Linter"
-    
-  - command: "source ~/.cargo/env && cargo test"
-    label: "Tests"
-
-  - command: "fossa init && fossa analyze --server-scan && fossa test"
-    label: "Fossa"
```
Comment (frol) [inline .buildkite/pipeline.yml:12]: Why do you believe it is not needed?

**Review by frol — CHANGES_REQUESTED** [review]

File: .buildkite/pipeline.yml (hunk 2 above)
Comment (chefsale) [reply .buildkite/pipeline.yml:12]: @mhalambek we cannot delete this.

Comment (chefsale) [reply .buildkite/pipeline.yml:12]: I believe that @mhalambek wanted to add this as a seperate pipeline, but didnt realized there was already this config. Rename the new one to docker-release.yml and leave the original one as is.

**Review by chefsale — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (mhalambek) [reply Dockerfile:28]: Did some experiments with it, it's not as easy, there are some runtime libs it requires, I've tried it using musl but it wasn't working properly

File: .buildkite/pipeline.yml (hunk 2 above)
Comment (mhalambek) [reply .buildkite/pipeline.yml]: I've deleted this by mistake, forgot to check if there is already something there

**Comment by frol** [comment]
@mhalambek do you still want to include the docker image build in the `.buildkite/pipeline.yml`? It is gone now

**Review by frol — CHANGES_REQUESTED** [review]
@mhalambek Just a friendly reminder ^

File: Dockerfile (hunk 1 above)
Comment (khorolets) [reply Dockerfile:28]: It needs only `libpq-dev` if I'm not mistaken

**Comment by frol** [comment]
@mhalambek I see a force-push with no changes. Was it intentional?

**Pull request closed without merging**